uk_bin_collection/uk_bin_collection/councils/tests/test_south_kesteven_district_council.py

```
@@ -1,249 +1,316 @@
+def council():
+    return CouncilClass()
+
+
+def make_option(text: str) -> MagicMock:
+    option = MagicMock()
+    option.text = text
+    return option
+
+
+def test_parse_data_requires_postcode(council):
+    with pytest.raises(ValueError, match="Postcode is required for South Kesteven."):
```

Ruff (0.15.15)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)